concourse/scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
+
+env=${DEPLOY_ENV-$1}
+pipeline="cf-deploy"
+config="${SCRIPT_DIR}/../pipelines/deploy.yml"
+
+[[ -z "${env}" ]] && echo "Must provide environment name" && exit 100
+bash "${SCRIPT_DIR}/deploy-pipeline.sh" "${env}" "${pipeline}" "${config}"
+
+
+export ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+export fly_target=${FLY_TARGET:-tutorial}
+
+fly unpause-pipeline --pipeline "${pipeline}"
+curl "${ATC_URL}/pipelines/${pipeline}/jobs/init-bucket/builds" -X POST
```

Comment:
Do we really want to trigger the pipeline whenever we deploy it? It can be risky/undesirable.

Additionally:

 * the task can be triggered several times concurrently if you run the deploy script several time. In this case does not matter but it can be an issue. I am not sure how to prevent that. 

 * I think the intention of the next command `fly watch -j ${pipeline}/vpc` is wait for the current triggered pipeline. But actually it will watch the latest finished job, which does not need to be (and it never is) the one triggered in this execution of `deploy.sh`, but the previous one.